scripts/package-json-cullify.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
i hate it when you want to cullify but you actually just omitify 🤷‍♂️

Reply:
Don't follow.

**cull**: (verb)
1. to select from a group : choose
// culled the best passages from the poet's work
// Damaged fruits are culled before the produce is shipped.
2. to reduce or control the size of (something, such as a herd) by removal (as by hunting) of especially weaker animals 
// The town issued hunting licenses in order to cull the deer population.